src/mmw/apps/monitoring/views.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from rest_framework import status
+
+from django.http import JsonResponse
```

Comment:
It is worth noting that I originally went with the DRF approach for this endpoint, but then switched to JsonResponse because I wanted to avoid the overhead. If there are reasons that approach might be better, feel free to suggest them.